src/components/demos/write/LoginForm.js

```diff
@@ -37,17 +36,21 @@ export default function LoginForm({
   const passwordAtMaxLength = passwordInput.length >= passwordUpperBound
 
   const enterUsernameAndPassword = async (e) => {
-    setApiOutput('')
```

Comment:
why remove? This stops login failed text from a previous attempt from clearing when you try again